keys.go

```diff
@@ -87,7 +87,7 @@ func CreateKey(name string) ([]byte, []byte, error) {
 
 func SaveKey(k []byte, file string) error {
 	os.MkdirAll(filepath.Dir(file), os.ModePerm)
-	err := os.WriteFile(file, k, 0644)
+	err := os.WriteFile(file, k, 0640)
```

Comment:
Why not `600`?

Reply:
I'd say 0600 is better since we do not run `sbctl` as it's own user, thus there is no use to have the root group readable I think?

Reply:
I actually pondered the same question before deciding on these permissions. My reasoning for `0640` was, that there might be edge cases on some systems which use the `root` group for non-root administrative users.
That being said, `0600` would also have been my first choice without these considerations and I'm happy to change the permissions to be even more restrictive and update the PR.

By the way, what's your opinion on making them even `0400` to gain some safety against accidental deletion? Those keys are somewhat precious after all :)